packages/roosterjs-editor-api/lib/format/clearFormat.ts

```diff
@@ -46,10 +46,18 @@ export default function clearFormat(editor: Editor) {
                 setFontSize(editor, defaultFormat.fontSize);
             }
             if (defaultFormat.textColor) {
-                setTextColor(editor, defaultFormat.textColor);
+                if (defaultFormat.textColors) {
+                    setTextColor(editor, defaultFormat.textColors);
+                } else {

```

Comment:
Can this logic be improved in this way?

```typescript
if (defaultFormat.textColor) {
  setTextColor(editor, defaultFormat.textColor);
}
else if (defaultFormat.textColors) {
  setTextColor(editor, defaultFormat.textColors);
} 
```

I think we don't need to set `textColor` if we are using `textColors`.